### DISCUSSION TITLE
How to change between 12 and 24 hour time display?

### DISCUSSION THREAD

**Discussion by 1920x1200** [zachowj/node-red-contrib-home-assistant-websocket#339] [Q&A]
Hello, is there a way to change these history messages into 12 hour (AM/PM) time format?

<img width="255" alt="Screen Shot 2021-01-18 at 6 28 45 PM" src="https://user-images.githubusercontent.com/38740977/105120369-1e076380-5aa0-11eb-94ca-103987c31f57.png">

I've searched for a solution to this and couldn't find anything. I posted on the Node-RED forums and they directed me over here as apparently nodes like "call service" shown above are specific to this integration.

Thanks,

Scott

**Comment by zachowj** [comment]
There currently isn't a way to change the status output. It is on a roadmap list somewhere.

You can edit the code directly and change one line and it should work. 

Change `false` to `true`: https://github.com/zachowj/node-red-contrib-home-assistant-websocket/blob/28928b4c8a6571c247b043781db0691000c535ed/lib/base-node.js#L273

**Comment by 1920x1200** [comment]
Awesome, thanks for taking a look at this and pointing me in the right direction. 
I'm assuming I could then just disable the default one, and use the newly modified one instead.
Once I change that value, how do I upload the now modified version to the pallet? 

Thanks,

Scott
